src/App.js

```diff
@@ -1,12 +1,89 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import './App.css';
+import axios from "axios";
+import PokemonInfo from "./components/PokemonInfo";
+import Button from "./components/Button";
+import pokemonlogo from "./assets/pokemonlogo.svg"
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    //Store the data in the state
+    const [allPokeData, setAllPokeData] = useState();
+    //Store the url in the state
+    const [pokeUrl, setPokeUrl] = useState(`https://pokeapi.co/api/v2/pokemon`);
+    //Store the disabled-state of the next-button
+    const [nextDisabled, setNextDisabled] = useState(false);
+    //Store the disabled-state of the previous-button
+    const [prevDisabled, setPrevDisabled] = useState(false);
+    //store the error-state
+    const [error, setError] = useState();
+
+    //nextbutton triggers nextclick function, previousbutton set to disabled
+    function nextClick() {
+        setPokeUrl(allPokeData.next);
+        setPrevDisabled(false);
```

Comment:
What if I go to the next page and back. Than the previous button would be enabled but won't work.